CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+# v0.12.0
+
+## Added
+
+- [[#187](https://github.com/rust-vmm/kvm-ioctls/pull/187)] Support for 
+  `KVM_SET_IDENTITY_MAP_ADDR`
+- Derive Debug for all exported structs and enums
+- [[#189](https://github.com/rust-vmm/kvm-ioctls/pull/189)] Expose `KVM_SET_` and 
+  `KVM_HAS_DEVICE_ATTR` for vcpu
+- [[#191](https://github.com/rust-vmm/kvm-ioctls/pull/191)] Add `KVM_TRANSLATE` and
```

Comment:
I would add more details here: ".. that allows translating guest virtual address to the physical address"

Reply:
Done